indico/modules/receipts/default_templates/attendance/metadata.yaml

```diff
@@ -0,0 +1,83 @@
+custom_fields:
+  - name: address_from
+    type: textarea
+    attributes:
+      label: Organizer Address
```

Comment:
Please check `indico/modules/receipts/default_templates/receipt/metadata.yaml` and copy the descriptions from there, and check if anything is missing (e.g. `add_affiliation`, not sure about the registrant's address + the corresponding override option)